src/app/payment/paymentMethod/getPaymentMethodDisplayName.spec.tsx

```diff
@@ -1,7 +1,7 @@
 import { createLanguageService, LanguageService } from '@bigcommerce/checkout-sdk';
 
 import { FALLBACK_TRANSLATIONS } from '../../locale/translations';
-import { getPaymentMethod } from '../payment-methods.mock';
+import { getPaymentMethod, getPaypalCrteditPaymentMethod } from '../payment-methods.mock';
```

Comment:
Correct misspelling pls `getPaypalCrteditPaymentMethod` =>  `getPaypalCreditPaymentMethod` and in other places too